corehq/apps/reports/models.py

```diff
@@ -80,16 +80,18 @@ class HQUserType(object):
     COMMTRACK = 4
     DEACTIVATED = 5
     WEB = 6
+    GROUPS_AND_LOCATIONS = 7
```

Comment:
nit: This isn't really a type of user, it's a filter option, more analogous to "All Data" or "Project Data".  What would you think of defining it like those?